### ISSUE TITLE
Suggestion: Skip files

### ISSUE THREAD

**Issue by AldebaraanMKII** [elvis972602/Kemono-scraper#48]
It seems the program downloads both the images and the .zip file which contains all the images inside. I think it is a waste of time to download both since they are the same thing in most cases. A option to download only one of these would be welcome.

**Comment by Shikitsuna** [comment]
agreed, if possible add word filter options too beside extension so you can filter something like WIP,PSD etc
